index.html

```diff
@@ -93,14 +93,20 @@ <h1 class="teamHeader">Team</h1>
     </section>
 
     <section id="contact">
-      <h1 class="contactHeader">Contact</h1>
-      <p>This first tag is almost the same as any other HTML tag you would
-        use to create a link to another page. However, to create a bookmark
-        anchor), you must start the link with a # (pound symbol,) which
-        represents a name statement, used in the next tag. When the user
-        clicks on the "Top" link, the computer would then go to the name tag,
-        if found on the same page of the link.
-      </p>
+
+      <h1>Contact</h1>
+      <div class="contact">
```

Comment:
Evrey one (team,contact,about) should be have a div , because it easy to control for all them ,, in the css if we want to do style it easy to do .